CS: clean up after merges

jrfnl opened the pull request:
## Summary

This PR can be summarized in the following changelog entry:

* _N/A_

## Relevant technical choices:


### Rename a new class to comply with the naming conventions.

As class names in PHP are case-insensitive, this should not be considered a BC-break.

### Clean up after merges

* Various whitespace fixes
* Multi-item keyed arrays should be multi-line
* Array alignment & comma after last item in multi-line array
* Missing translation comments
* Escape one non-direct output
* Minor comment fixes



## Test instructions

This PR can be tested by following these steps:

* _N/A_

moorscode commented on admin/views/tabs/metas/post-types.php:
```diff
@@ -42,12 +42,13 @@
 
 		$toggle_icon = 'dashicons-arrow-up-alt2';
 		$class       = 'toggleable-container';
+		$class       = 'toggleable-container';
```
This line seems duplicated.

jrfnl commented:
I've rebased the PR which got rid of the duplicate line.

I've also done a quick rescan and fixed some issues introduced since I pulled this PR. The additional fixes are in a separate commit for ease of reviewing.
